frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -127,3 +132,43 @@ export const getJoinedCardsDataset = (
 
   return Array.from(groupedData.values());
 };
+
+/**
+ * Sorts the series models based on the order specified in the "graph.series_order" property of visualization settings.
+ * It defines the breakout series order so that charts where all series are metrics without breakouts do not
+ * rely on this property, and the series are sorted based on the metrics order in the original MBQL query.
+ *
+ * @param {SeriesModel[]} seriesModels - The array of series models to be sorted.
+ * @param {ComputedVisualizationSettings} settings - The computed visualization settings.
+ * @returns {SeriesModel[]} The sorted array of series models.
+ */
+export const getSortedSeriesModels = (
```

Comment:
Nit regarding nomenclature: I think "ordered" is better than "sorted" here (e.g. `getOrderedSeriesModels`. "Sort" is ambiguous with sorting in the MBQL query, and what the user is doing here isn't sorting in ascending or descending order, but rather defining their own unique order, which is different from sorting.